eng/build.ps1

```diff
@@ -20,46 +20,78 @@ Param(
 
 function Get-Help() {
   Write-Host "Common settings:"
-  Write-Host "  -subset                   Build a subset, print available subsets with -subset help (short: -s)"
-  Write-Host "  -vs                       Open the solution with VS using the locally acquired SDK. Path or solution name (ie -vs Microsoft.CSharp)"
-  Write-Host "  -os                       Build operating system: Windows_NT, Linux, OSX, or Browser"
-  Write-Host "  -arch                     Build platform: x86, x64, arm, arm64, or wasm (short: -a). Pass a comma-separated list to build for multiple architectures."
-  Write-Host "  -configuration            Build configuration: Debug, Release or [CoreCLR]Checked (short: -c). Pass a comma-separated list to build for multiple configurations"
-  Write-Host "  -runtimeConfiguration     Runtime build configuration: Debug, Release or [CoreCLR]Checked (short: -rc)"
-  Write-Host "  -librariesConfiguration   Libraries build configuration: Debug or Release (short: -lc)"
-  Write-Host "  -verbosity                MSBuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
-  Write-Host "  -binaryLog                Output binary log (short: -bl)"
-  Write-Host "  -help                     Print help and exit (short: -h)"
+  Write-Host "  -arch (-a)                     Build platform: x86, x64, arm, arm64, or wasm."
+  Write-Host "                                 Pass a comma-separated list to build for multiple architectures."
+  Write-Host "                                 [Default: Your machine's architecture.]"
+  Write-Host "  -binaryLog (-bl)               Output binary log."
+  Write-Host "  -configuration (-c)            Build configuration: Debug, Release or [CoreCLR]Checked."
```

Comment:
What is `CoreCLRChecked` ? I can't see references to it in any of our docs. How is it different to using `-runtimeConfiguration`

Ah -- you mean `Checked`, but it only works when building coreclr? Then could it say
> ... Release, or Checked. (Checked only applicable to coreclr)

Reply:
same below for -runtimeConfiguration

Reply:
You might also briefly note the difference between Checked and Debug. (ie., in both cases, code contains asserts and DEBUG is defined [I think?] but Debug is non-optmized)

Reply:
> Ah -- you mean Checked, but it only works when building coreclr? Then could it say

right, that was me trying to express that :D